<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-O5SVODT" failures="0" tests="2" name="com.rahul.hpmgmt.services.impl.ExceptionTest" time="0.035" errors="0" timestamp="2019-08-26T16:42:12 IST" skipped="0">
  <testcase classname="com.rahul.hpmgmt.services.impl.ExceptionTest" name="patientWithIdNotFoundExceptionTest" time="0.005"/>
  <testcase classname="com.rahul.hpmgmt.services.impl.ExceptionTest" name="idAlreadyExistsExceptionTest" time="0.030"/>
</testsuite> <!-- com.rahul.hpmgmt.services.impl.ExceptionTest -->
